bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -7,6 +7,7 @@ from airflow.utils.task_group import TaskGroup
 import datetime
 from utils.constants import ALLOWED_STATES, FAILED_STATES
 from utils.gcp import bigquery_etl_query, gke_command, bigquery_dq_check
+from utils.slack import send_slack_notification
```

Comment:
nit: is there a convenient jinja way to do something like `if any(task.slack_notification['status'] for task in tasks)` here? Or upstream in the render